### PR TITLE
test(WebexMeeting): e2e test waiting for others after join

### PR DESCRIPTION
**Pull request by karinasigartau0798**
Create a E2E Test to Assert User Waits for Others After Joining a Meeting via Meetings Widget.

https://jira-eng-gpk2.cisco.com/jira/secure/Dashboard.jspa

### CODE REVIEW

File: tests/WebexMeeting.e2e.js [hunk 1]
```diff
@@ -47,4 +47,12 @@ describe('Meeting Widget', () => {
     MeetingPage.loadWidget(meetingDestination);
     expect(MeetingPage.meetingInfo).toBeDisplayed();
   });
+
+  it('display waiting for others after joining meeting', () => {
```
Comment (cipak) [inline tests/WebexMeeting.e2e.js:51]: 3rd person:
```sh
Meeting Widget (line 5)
- has...
- loads...
- displays "Waiting for others" after joining meeting
- ```

File: tests/WebexMeeting.e2e.js [hunk 2]
```diff
@@ -47,4 +47,12 @@ describe('Meeting Widget', () => {
...
+    MeetingPage.loadWidget(meetingDestination);
+    expect(MeetingPage.waitingForOthers).not.toExist();
+    MeetingPage.joinMeetingBtn.click();
+    MeetingPage.waitForWaitingForOthersScreen();
+    expect(MeetingPage.waitingForOthers).toExist();
```
Comment (cipak) [inline tests/WebexMeeting.e2e.js:56]: maybe it exists but it's not visible, let's use `toBeVisible` to be sure

File: tests/pages/MeetingWidget.page.js [hunk 3]
```diff
@@ -3,12 +3,21 @@ class MeetingWidgetPage {
...
+  get waitingForOthers() { return $('h4=Waiting for others to join...'); }
 
   loadWidget(meetingDestination) {
     this.destination.setValue(meetingDestination);
     this.displayWidgetBtn.click();
     this.interstitialMeeting.waitForDisplayed({timeout: 30000});
   }
+
+  waitForWaitingForOthersScreen() {
+    browser.waitUntil(() => this.waitingForOthers, {
```
Comment (cipak) [inline tests/pages/MeetingWidget.page.js:16]: `this.waitingForOthers` is always truthy. Please use https://webdriver.io/docs/api/element/waitForDisplayed/ like here: https://github.com/webex/widgets/pull/54/files#diff-063a925e621a59156c6d99c3a82c63ed56563e2235370893ee1c8699dbe6f542R12

File: tests/WebexMeeting.e2e.js [hunk 4]
```diff
@@ -47,4 +47,12 @@ describe('Meeting Widget', () => {
...
     expect(MeetingPage.meetingInfo).toBeDisplayed();
   });
+
+  it('displays "Waiting for others" after joining meeting', () => {
+    MeetingPage.loadWidget(meetingDestination);
+    expect(MeetingPage.waitingForOthers).not.toExist();
+    MeetingPage.joinMeetingBtn.click();
+    MeetingPage.waitingForOthers.waitForDisplayed({timeout: 10000});
+    expect(MeetingPage.waitingForOthers).toBeVisible();
+  });
```
Comment (cipak) [inline tests/WebexMeeting.e2e.js:57]: +unloadWidget

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**